### ISSUE TITLE
fail

### ISSUE THREAD

**Issue by buwads288** [lee-1980-jun/TikTok-bot#1]
![1](https://github.com/lee-1980-jun/TikTok-bot/assets/146057177/ee78dd23-6782-4f70-8498-266d9fcc47bf)

**Comment by anoleido** [comment]
Yep it dose not work, on line 71 req.text give 403 forbidden error requests
